src/style.css

```diff
@@ -1,4 +1,9 @@
 /* ROOT ELEMENTS */
+* {
+  margin: 0;
```

Comment:
is this a good idea to set margin and padding to 0 right away ?

Reply:
I feels like it gives more clear view and control by overriding browser default. Also leaving broswer default would leave header looking out off place.